RealmBrowser/Classes/RLMViewController.m

```diff
@@ -71,9 +71,14 @@ - (void)setSelectionIndex:(NSUInteger)newIndex
 {
     NSUInteger oldIndex = self.tableView.selectedRow;
     if (oldIndex != newIndex) {
-        [self.tableView selectRowIndexes:[NSIndexSet indexSetWithIndex:newIndex] byExtendingSelection:NO];
-        
-        [self.tableView scrollRowToVisible:newIndex];
+        if (newIndex == (NSUInteger)-1) {
```

Comment:
It might be more appropriate to use `NSUIntegerMax` instead of typecasting -1 there. :)